kubernetes-1.31.yaml

```diff
@@ -62,11 +62,6 @@ pipeline:
 
       make WHAT="$WHAT"
 
-  - runs: |
```

Comment:
This may introduce a regression for unprivileged deployments that relied on the capability on the binary file to be able to run on the host network and bind on the usual privileged port 6443.

Nevertheless, I think it's a good refactoring because this way we are giving least privilege by default.
For cases where the API server will be exposed on unprivileged ports (>1024), this capability isn't needed at all.
In cases where it is, the capability can be added at runtime at thread level.

Reply:
But I think we should test these scenarios before introducing this change.

Reply:
I checked and the extended attribute (`security.capabilities`) is not persisted, so this step is not changing anything in the output apk file.

I am confident it is safe to remove it from the `kubernetes` package.

Reply:
Good catch! That's unexpected, I'd say